integration_test/scripts/publish-images.sh

```diff
@@ -1,14 +1,37 @@
 #!/bin/bash
-# imageProject publishFlag
+# imageProject publishFlag repositoryString
+
+accessToken=$ACCESS_TOKEN
+repositoryString=$3
+IFS='_' read -r -a repoArray <<< "$repositoryString"
+githubUser="${repoArray[1]}"
+githubRepo="${repoArray[2]}"
+if [ -z "$githubUser" ]
```

Comment:
Why would it be empty?

Reply:
https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/pull/63#discussion_r399173920
i may remove this part but if some user will fork repo releases wont work

Reply:
>In the case of $repositoryString only containing "repoName", wouldn't $githubRepo be empty instead of $githubUser since it's the second entry in the array?

since $repositoryString is not contains any "_", all entries of array are empty

> I guess we can make the organization github user a substitution to get around this

but we'll have to keep parse REPO_NAME, or replace name of the repo with substitution as well

Reply:
Thanks for the explanation.

Well, I guess this build file is already project specific due to the encrypted key we define below. Maybe we can give up on making this work for fork repos, unless someone asks for that feature.